aspy/refactor_imports/classify.py

```diff
@@ -110,7 +110,10 @@ def _get_module_info(module_name, application_dirs):
         elif spec.origin in {None, 'namespace'}:
             return True, next(iter(spec.submodule_search_locations)), False
         # special case pypy3 bug(?)
```

Comment:
this comment has been orphaned

Reply:
Removed orphaned comment.  Let me know if you instead wanted me to try and re-hang it somewhere in the enlarged logic.

Reply:
I think I meant more that this _was_ pointing out a case where there was a bug in pypy3 previously, but now that maybe isn't being caught 🤔 -- would have to dig through the commit history (and probably try that version of pypy3 again against the new logic)

Reply:
CI is still running against pypy3, so I had assumed that the original edge case was still being handled correctly.